ble-shelly-blu.js

```diff
@@ -6,87 +6,62 @@
  * parsed from the BLE device
  *
- * All paramerts that the BLU devices can return:
- * Each device will provide data solely from its sensors.
- * - pid - packet ID
- * - battery - the battery level of the device in %
- * - temperature - the temperature value in °C if the device has temperature sensor
- * - humidity - the himidity value in % if the device has humidity sensor
- * - illuminance - the illuminance value in lux if the device has light sensor
- * - motion - 0/1 (motion/clear) if the device has motion sensor
- * - window - 0/1 (close/open) if the device has reed switch
- * - button - the number of presses if the device has button
- * - rotation - the angle of rotatation in ° if the device has gyroscope
- * - rssi - the signal strength is dB
- * - address - The mac address of the Shelly BLU device
+ * Represents data provided by each device.
+ * Every value illustrating a sensor reading (e.g., button) may be a singular sensor value or 
+ * an array of values if the object has multiple instances. 
+ * 
+ * @typedef {Object} DeviceData
+ * @property {number} pid - Packet ID.
+ * @property {number} battery - The battery level of the device in percentage (%).
+ * @property {number} rssi - The signal strength in decibels (dB).
+ * @property {string} address - The MAC address of the Shelly BLU device.
+ * @property {string} model - The model of the Shelly BLU device.
+ * @property {number | number[]} [temperature] - The temperature value in degrees Celsius if the device has a temperature sensor.
+ * @property {number | number[]} [humidity] - The humidity value in percentage (%) if the device has a humidity sensor.
+ * @property {number | number[]} [illuminance] - The illuminance value in lux if the device has a light sensor.
+ * @property {number | number[]} [motion] - Motion status: 0 for clear, 1 for motion (if the device has a motion sensor).
+ * @property {number | number[]} [window] - Window status: 0 for closed, 1 for open (if the device has a reed switch).
+ * @property {number | number[]} [button] - The number of presses if the device has a button.
+ * @property {number | number[]} [rotation] - The angle of rotation in degrees if the device has a gyroscope.
  * 
  * Example event data: {"component":"script:*","name":"script","id":*,"now":*,"info":{"component":"script:*","id":*,"event":"shelly-blu","data":{"encryption":false,"BTHome_version":2,"pid":118,"battery":100,"button":1,"rssi":-76,"address":*},"ts":*}}
  */
 
 /******************* START CHANGE HERE *******************/
-let CONFIG = {
+const CONFIG = {
   // Specify the destination event where the decoded BLE data will be emitted. It allows for easy identification by other applications/scripts
   eventName: "shelly-blu",
 
-  // When set to true, debug messages will be logged to the console
-  debug: false,
-
-  //When set to true and the script ownes the scanner, the scan will be active. 
+  //When set to true and the script owns the scanner, the scan will be active. 
```

Comment:
If this script owns the scanner then selecting active or passive is possible, otherwise the option the scanner was started with will be used